### PR TITLE
Ensure that stratisd builds against 1.24.1

### PR DESCRIPTION
**Pull request by agrover**
This is what is in Debian stable.

Signed-off-by: Andy Grover <agrover@redhat.com>

### CODE REVIEW

**Comment by mulkieran** [comment]
Failed mkfs.ext3 failure, redoing.

**Comment by agrover** [comment]
seems to work! trivial?

File: .travis.yml [hunk 1]
```diff
@@ -30,7 +30,7 @@ matrix:
         - rust: stable
           env: TASK=build TARGET=x86_64-unknown-linux-gnu
           install:
-            - rustup default 1.25.0
+            - rustup default 1.24.1
```
Comment (mulkieran) [inline .travis.yml]: You should fix the comment above the target.

**Comment by agrover** [comment]
You mean moving to using features in more recent Rust releases? Absolutely. But just driven by the oldest distro release we want to (someday) be supported in, which is Debian stable at the moment.

**Comment by agrover** [comment]
Meh, not going to bother this this. closing.

**Pull request closed without merging**